src/thermo/MixtureFugacityTP.cpp

```diff
@@ -44,6 +47,25 @@ int MixtureFugacityTP::reportSolnBranchActual() const
     return iState_;
 }
 
+// ---- Molar Thermodynamic Properties ---------------------------
+double MixtureFugacityTP::enthalpy_mole() const
+{
+   _updateReferenceStateThermo();
```

Comment:
If `setTemperature()` calls `_updateReferenceStateThermo()`, is it necessary to call it anywhere else? I think most of these calls can be eliminated.

Either that, or it should not be called in `setTemperature` and does need to be called before every function that accesses the reference state thermo. This is effectively the approach used in `IdealGasPhase`, but I'm not necessarily advocating it.

Reply:
Removed this call from the other thermodynamic functions.